src/Function/Equality.agda

```diff
@@ -11,6 +11,7 @@
 -- `Bijection`. The alternative definitions found in this file will
 -- eventually be deprecated.
 
+-- check if modules are actually using Function.Equality and port them
```

Comment:
Could we get rid of these temporary notes before merging?

Reply:
Oh oops, those are my notes, pushed them by mistake. Thanks!